source/globalCommands.py

```diff
@@ -3260,6 +3264,31 @@ def script_braille_cycleCursorShape(self, gesture):
 		# Translators: Reports which braille cursor shape is activated.
 		ui.message(_("Braille cursor %s") % shapeMsg)
 
+	@script(
+		# Translators: Input help mode message for toggle braille show messages command.
+		description=_("Cycle through the braille show messages modes"),
+		category=SCRCAT_BRAILLE
+	)
+	def script_braille_cycleShowMessages(self, gesture):

```

Comment:
Type hints have usually been used for the last added scripts in this file and more generally by new functions in NVDA code.

```suggestion
	def script_braille_cycleShowMessages(self, gesture: inputCore.InputGesture) -> None:
```